src/Views/gagepage.html

```diff
@@ -224,7 +224,7 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                     <div ng-if="!vm.chartConfig" class="panel panel-default reportSpinner">
                         &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
                     </div>
-                    <center>
+                    <!-- <center>
```

Comment:
chart1 and chart2 are now commented out, so only the Stage vs. Discharge plot is visible:

![image](https://user-images.githubusercontent.com/40237491/226744460-4906e1bb-2267-4784-93bb-6411870d6232.png)